app/templates/partials/answer-guidance.html

```diff
@@ -15,11 +15,13 @@
 
 {% set guidance_body = {
   "id": "guidance-" ~ answer_guidance.id ~ "-body",
+  "id": "guidance-" ~ answer_guidance.id ~ "-body",
   "aria-hidden": "true"
 } %}
 
 <div class="guidance js-details" {{guidance|xmlattr}}>
-  <a class="guidance__link js-details-trigger js-details-label mars" {{guidance_link|xmlattr}}>{{_("Show further guidance")}}</a>
+  <a class="guidance__link js-details-trigger js-details-label mars"
+    {{guidance_link|xmlattr}} {{helpers.track('click', 'Navigation', 'Guidance', answer_guidance.id)}}>{{_("Show further guidance")}}</a>
```

Comment:
Maybe the category for this should be something else; 'Help' or 'Content' or something?